src/ngraph/op/util/attr_types.hpp

```diff
@@ -49,5 +51,33 @@ namespace ngraph
             AUTO = SAME_UPPER,
             NOTSET = EXPLICIT
         };
+
+        enum class AutoBroadcastType
+        {
+            NONE = 0,
+            NUMPY
```

Comment:
It is a shortcut to just say that this works like numpy broadcast, but it would be nice to actually know the rules without having to go find numpy's documentation to find out how our code works.